testing/MLDB-180-basic-join.js

```diff
@@ -258,7 +258,7 @@ for (var i in funcs) {
 // MLDB-1088
 
 expected = [
-   [ "_rowName", "test1.x", "test1.y", "test1.z" ],
+   [ "_rowName", "x", "y", "z" ],
```

Comment:
OK, final answer (until the next change) ... we should keep the prefixes for all joins